packages/rn-tester/NativeComponentExample/js/MyNativeView.js

```diff
@@ -85,6 +85,14 @@ function getTextFor(measureStruct: MeasureStruct): string {
     measureStruct.height,
   )}`;
 }
+const opacityDecrementCounter = 0.2;
+
+function getOpacity(opacity: number): number {
```

Comment:
```suggestion
function computeNextOpacity(opacity: number): number {
```
Can we change the name of the function to make clearer what's its purpose? Otherwise it looks like it should return the opacity value, but it is actually doing much more.

Reply:
I have updated the function name as you said. Please review